src/components/Cast/CastHeader.tsx

```diff
@@ -0,0 +1,28 @@
+import { HeaderText, LinkText, SubHeaderText } from 'components/ui/Text'
+import { displayFrom } from 'helpers/visibilityClassnames'
+import classnames, { display, space, width } from 'classnames/tailwind'
+
+export default function () {
+  const farcasterEchoUrl =
+    'farcaster://profiles/0xAD81Fb97FEcaFA3A9830d4e2F424aC1776024DA8/posts'
+
+  return (
+    <div
+      className={classnames(space('space-y-4'), width('md:w-full', 'w-4/5'))}
```

Comment:
Let's extract `classnames` things into `const`s, it's better to have them separated

Reply:
Also, how this behaves on mobiles? Maybe it's better to use `flex` + `gap`?

Reply:
fixed, ty